connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -154,7 +154,14 @@ export async function githubUpsertIssueActivity(
     documentUrl: issue.url,
     timestampMs: lastUpdateTimestamp,
     tags: tags,
-    parents: [],
+    // The convention for parents is to use the external id string; it is ok for
+    // repos, but not practical for issues since the external id is the
+    // issue number, which is not guaranteed unique in the workspace.
+    // Therefore as a special case we use getIssueDocumentId() to get a parent string
+    parents: [
+      getIssueDocumentId(repoId.toString(), issue.number),
+      repoId.toString(),
```

Comment:
Just out of curiosity what does this Id look like? Is that a number?

Reply:
If we have 2 github managed data sources I worry we'll be clashing, which means we should probably update the connectors resource API to use something more specific?

We want to make sure it's globally unique

Reply:
No clear doc about this, but I have created two repos at ~ 30mn interval and their repo ids are 688834560 and 688870568
Pretty sure it's an autoincremented primary key
=> I am willing to decide it is globally unique, and be responsible for the consequences if i'm wrong

Reply:
These look long enough to me :+1: :)